scripts/js/lib/links/ignores.ts

```diff
@@ -79,6 +68,52 @@ export const ALWAYS_IGNORED_URLS = new Set([
   ...ALWAYS_IGNORED_URLS__SHOULD_FIX,
 ]);
 
+// -----------------------------------------------------------------------------------
+// Always ignored URL regexes - be careful using this
+// -----------------------------------------------------------------------------------
+
+function _addonsObjectsInvRegexes(): string[] {
+  // Addons have non-API docs in their Sphinx build that translate into invalid links
+  // we should ignore
+  return ["how-tos", "how_tos", "install", "index", "explanations"]
+    .map((path) => [
+      // Latest version
+      `\/api\/qiskit-addon-[^\/]+\/${path}(\/.*|#.*|$)`,
+      // Historical versions
+      `\/api\/qiskit-addon-[^\/]+\/[0-9]+\.[0-9]\/${path}(\/.*|#.*|$)`,
+      // Dev version
+      `\/api\/qiskit-addon-[^\/]+\/dev\/${path}(\/.*|#.*|$)`,
+    ])
+    .flat();
+}
+
+function _runtimeObjectsInvRegexes(): string[] {
+  // Runtime has non-API docs in their Sphinx build that translate into invalid links
+  // we should ignore
+  return [
+    "errors",
+    "migrate",
+    "cloud",
+    "faqs",
+    "index",
+    "sessions",
+    "primitives",
+    "compare",
+    "retired",
+  ]
+    .map((path) => [
+      `\/api\/qiskit-ibm-runtime\/0.16\/${path}(\/.*|#.*|$)`,
+      `\/api\/qiskit-ibm-runtime\/0.15\/${path}(\/.*|#.*|$)`,
+      `\/api\/qiskit-ibm-runtime\/0.14\/${path}(\/.*|#.*|$)`,
+    ])
+    .flat();
```

Comment:
Ditto flatmap

Reply:
I removed the `flat` call in this case given that it's not necessary anymore after combining the three regexes into one